<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addurlDialog</class>
 <widget class="QDialog" name="addurlDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>507</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>782</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>782</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加起始URL</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>moon.ico</normaloff>moon.ico</iconset>
  </property>
  <widget class="QTabWidget" name="addurltabWidget">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>19</y>
     <width>731</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="singlesitetab">
    <attribute name="title">
     <string>单条网址</string>
    </attribute>
    <widget class="QPushButton" name="singleaddButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="singleaddlabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>逐条输入网址：（一行一个）</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="singleaddplainTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>661</width>
       <height>181</height>
      </rect>
     </property>
     <property name="plainText">
      <string>http://</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="batchinputtab">
    <attribute name="title">
     <string>批量输入</string>
    </attribute>
    <widget class="QLabel" name="siteformatlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>地址格式：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="siteformatlineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
    <widget class="QPushButton" name="batchinputButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="geometricradioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>等比数列</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="alphabeticallyradioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>字母变化</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="arithmeticradioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>等差数列</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="arithmeticfirstitemlabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>首项</string>
     </property>
    </widget>
    <widget class="QLabel" name="geometricfirstitemlabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>首项</string>
     </property>
    </widget>
    <widget class="QLabel" name="alphabeticallyfromlabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>从</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="arithmeticfirstitemlineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="geometricfirstitemlineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="alphameticallyfromlineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
    <widget class="QLabel" name="arithmeticitemslabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>项数</string>
     </property>
    </widget>
    <widget class="QLabel" name="geometricitemslabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>项数</string>
     </property>
    </widget>
    <widget class="QLabel" name="alphameticallytolabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>到</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="arithmeticitemnumberlineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="geometricitemnumberlineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="alphameticallytolineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
    <widget class="QLabel" name="tolerancelabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>公差</string>
     </property>
    </widget>
    <widget class="QLabel" name="ratiolabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>公比</string>
     </property>
    </widget>
    <widget class="QLabel" name="casesensitivelabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>（区分大小写）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tolerancelineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ratiolineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="batchinputpreviewtextBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>671</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="batchinputpreviewlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>135</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>预览：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wildcardpushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="textimporttab">
    <attribute name="title">
     <string>文本导入</string>
    </attribute>
    <widget class="QLabel" name="textimportlocationlabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文本位置：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textimportlocationlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="textimportlocationpushButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="textimportpreviewlabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>预览：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="textimportButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>45</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textimportpreviewtextBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>75</y>
       <width>641</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="allurllabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全部URL：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="allurltextBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>721</width>
     <height>131</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
